grand_challenge_forge/partials/example-evaluation-method/evaluate.py.j2

```diff
@@ -1,19 +1,29 @@
 """
 The following is a simple example evaluation method.
 
-It is meant to run within a container.
+It is meant to run within a container. Its steps are as follows:
+
+  1. Read the algorithm output
+  2. Associate original algorithm input with a ground truth via predictions.json
+  3. Calculate metrics by comparing the algorithm output to the ground truth
+  4. Repeat steps 1, 2, and 3 for all algorithm outputs
```

Comment:
Step 4 could be misinterpreted. You're looping over jobs, where each job can have multiple outputs... so saying here that you loop over all outputs is not fully accurate. Maybe stick to what we call it on the platform "Repeat for all algorithm jobs that ran for this submission"?